cmd/agent/common/commands/check.go

```diff
@@ -122,6 +124,10 @@ func Check(loggerName config.LoggerName, confFilePath *string, flagNoColor *bool
 				color.NoColor = true
 			}
 
+			if generateIntegrationTraces {
+				os.Setenv("DDEV_TRACE_ENABLED", "true")
```

Comment:
Rather than env vars, is it possible to modify check config directly like we do for memory profiling?